src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -57,9 +57,9 @@ public void TestInterpolatedVerbatimString()
         public void TestAltInterpolatedVerbatimString_CSharp73()
         {
             UsingExpression(@"@$""hello""", TestOptions.Regular7_3,
-                // (1,1): error CS8401: To use '@$' instead of '$@' for an interpolated verbatim string, please use language version 8.0 or greater.
+                // (1,1): error CS8401: To use '@$' instead of '$@' for an interpolated verbatim string, please use language version preview or greater.
```

Comment:
>please use language version preview or greater. [](start = 105, length = 47)

same suggestion here and possible some other messages that use RequiredLanguageVersion